packages/dev/core/src/Rendering/prePassRenderer.ts

```diff
@@ -583,6 +613,37 @@ export class PrePassRenderer {
         return cfg;
     }
 
+    /**
+     * Retrieves an effect configuration by name
+     * @param name
+     * @returns
+     */
+    public getEffectConfiguration(name: string): PrePassEffectConfiguration | null {
+        for (let i = 0; i < this._effectConfigurations.length; i++) {
+            if (this._effectConfigurations[i].name === name) {
+                return this._effectConfigurations[i];
+            }
+        }
+
+        return null;
+    }
+
+    /**
+     * Keeps the sync between the noPrePassDefaultRT and the defaultRT
+     * @returns
+     */
+    private _syncPrePassRT() {
+        if (this.mrtCount === 0) {
+            return;
+        }
+        const needsInit = this.noPrePassDefaultRT.count !== 1;
+        if (needsInit) {
+            this.noPrePassDefaultRT.updateCount(1, { types: [this._mrtTypes[0]], formats: [this._mrtFormats[0]] });
+        }
+        this.noPrePassDefaultRT.setInternalTexture(this.defaultRT.renderTarget!.textures![0], 0, true);

```

Comment:
I forgot to comment on this one.

After the removal of `texture.incrementReferences();` in `RenderTargetWrapper.setTexture`, you will have to call `incrementReferences()` on `this.defaultRT.renderTarget!.textures![0]` yourself.